loadtest/user/userentity/actions.go

```diff
@@ -1091,3 +1091,16 @@ func (ue *UserEntity) UpdateConfig(cfg *model.Config) error {
 	ue.store.SetConfig(cfg)
 	return nil
 }
+
+// MessageExport creates a job for a compliance message export
+func (ue *UserEntity) MessageExport() error {
+	messageExportJob := &model.Job{
+		Type: "message_export",
```

Comment:
Do we want to set some data attributes? Or just go with the defaults?

Reply:
This is basically mimicking the actual API call that the webapp is doing when triggering a message export. So it only creates a new message export job with the `Data: nil`

Reply:
Perfect. :+1: